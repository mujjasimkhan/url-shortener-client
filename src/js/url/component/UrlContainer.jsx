import React from "react";
import { Container } from 'react-bootstrap';
import "../style/App.css";
import UrlForm from "./UrlForm.jsx";

class UrlContainer extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    const { shortenedUrlResponse } = this.props;
    return (
      <Container className="url-shortener-container-border">
        <UrlForm />
      </Container>
    );
  }
}

export default UrlContainer;