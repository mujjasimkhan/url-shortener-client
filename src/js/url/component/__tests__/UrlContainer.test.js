import React from 'react';
import renderer from 'react-test-renderer';
import UrlContainer from '../UrlContainer.jsx';

it('renders Url container components', () => {
  const tree = renderer
    .create(<UrlContainer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});