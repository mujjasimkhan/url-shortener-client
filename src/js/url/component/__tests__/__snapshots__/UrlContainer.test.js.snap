// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Url container components 1`] = `
<div
  className="url-shortener-container-border container"
>
  <div>
    <form
      className="url-shortener-form-padding"
      onSubmit={[Function]}
    >
      <div
        className="row"
      >
        <div
          className="col-lg-1 col"
        >
          <label
            className="form-label"
          >
            URL
          </label>
        </div>
        <div
          className="col"
        >
          <input
            className="form-control"
            onChange={[Function]}
            placeholder="Enter url"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-lg-1 col"
        />
        <div
          className="url-shortener-col-padding col-lg-2 col"
        >
          <input
            className="btn btn-primary"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
            type="submit"
            value="Submit"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;
